### PR TITLE
Small cleanup, partially in support of fc-app#11

### PR DESCRIPTION
**Pull request by jmthibault79**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by jmthibault79** [comment]
https://github.com/DataBiosphere/firecloud-app/issues/11

Subworkflows don't have Workflow Logs, so don't render a field when they're missing.

The rest is just my compulsive need to refactor.

File: src/cljs/main/broadfcui/page/workspace/monitor/common.cljs [hunk 1]
```diff
@@ -28,7 +28,7 @@
 (def call-failure-statuses #{"Failed" "Aborted"})
 
 (defn google-account-chooser-prefix [destination] (str "https://accounts.google.com/AccountChooser?continue=" destination))
-(def google-cloud-context (google-account-chooser-prefix "https://console.cloud.google.com/storage/browser/"))
+(def google-storage-context (google-account-chooser-prefix "https://console.cloud.google.com/storage/browser/"))
```
Comment (jmthibault79) [inline src/cljs/main/broadfcui/page/workspace/monitor/common.cljs:31]: "cloud" is too vague here

File: src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs [hunk 2]
```diff
@@ -235,26 +235,24 @@
...
+      (create-field "Started" (moncommon/render-date start)))
+    (when-let [end (workflow "end")]
+      (create-field "Ended" (moncommon/render-date end)))
     [IODetail {:label "Inputs" :data (utils/parse-json-string (get-in workflow ["submittedFiles", "inputs"]))}]
     [IODetail {:label "Outputs" :data (workflow "outputs")}]
-    [:div {:style {:whiteSpace "nowrap" :marginRight "0.5em"}}
-     (let [wlogurl (str "gs://" bucketName "/" submission-id "/workflow.logs/workflow."
-                        (workflow "id") ".log")]
-       (create-field "Workflow Log" (display-value wlogurl (str "workflow." (workflow "id") ".log"))))]
+    (when-let [workflowLog (workflow "workflowLog")]
```
Comment (jmthibault79) [inline src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs:254]: we don't need to construct this because Cromwell tells us directly

Comment (davidangb) [reply src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs:254]: this `workflowLog` also removes a `div` - intentional?

**Review by davidangb — APPROVED** [review]
:+1: with one small potential comment inline

File: src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs (hunk 2 above)
Comment (jmthibault79) [reply src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs:254]: Nope but ... it doesn't do anything.

**Comment by jmthibault79** [comment]
jenkins retest

**Pull request merged**